.github/workflows/base-version.txt

```diff
@@ -0,0 +1 @@
+0.9.1
```

Comment:
Why was this version value chosen?

Reply:
It's just a random value and can be set whatever we want to.